blackmarket/Home/Views/ProductViewCell.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+import SwiftUI
+
+class ProductViewCell: UICollectionViewCell {
+  
+  private lazy var favoriteImage = UIImageView()
+  private lazy var statusImage = UIImageView()
+  private lazy var title = UILabel.titleLabel(
+    text: "Baumler",
+    font: .h5Bold,
+    textColor: .labelTitle)
+  private lazy var productImage = UIImageView()
+  private lazy var price = UILabel.titleLabel(
+    text: "$36",
+    font: .h5Bold,
+    textColor: .labelTitle)
+  private lazy var cardView = UIView()
+  private lazy var divider = UIView()
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    
+    productImage.image = UIImage(named: "chair")
+    statusImage.image = UIImage(named: "restored_status")
```

Comment:
Looking at the image assets, this one can be easily achieved with a custom `UIView`.